WebDriverAgentLib/Categories/XCUIElement+FBPickerWheel.m

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "XCUIElement+FBPickerWheel.h"
+
+#import "FBRunLoopSpinner.h"
+
+@implementation XCUIElement (FBPickerWheel)
+
+- (BOOL)fb_scrollWithOffset:(CGFloat)relativeHeightOffset
+{
+  NSString *previousValue = [self value];
+  XCUICoordinate *startCoord = [self coordinateWithNormalizedOffset:CGVectorMake(0.5, 0.5)];
+  XCUICoordinate *endCoord = [startCoord coordinateWithOffset:CGVectorMake(0.0, relativeHeightOffset * self.frame.size.height)];
+  [endCoord tap];
+  NSError *error;
```

Comment:
This error is not passed anywhere. Please add this to `fb_select*` methods and handle at the endpoint properly